src/test/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpmultiline/InputRegexpMultilineMultilineSupport.java

```diff
@@ -19,7 +19,7 @@
  */
 public class InputRegexpMultilineMultilineSupport {
     void method() { // violation below
-// abc - violation
+// abc
```

Comment:
I think it would make more sense to remove the `violation below` and keep the single line `violation`.

Reply:
we matching regexp over comments, lets keep it, it is just a test.
`violation below` is used to not overlap comment marker with target code(code comments).

Reply:
Violation below text does not match `VIOLATION_BELOW_PATTERN` or `VIOLATION_BELOW_WITH_EXPLANATION_PATTERN` which brings up a point in a similar comment about why we aren't throwing an exception on wrong violation messages.

Reply:
No exception - Because we now has most aggressive match by word `violation` at any place.
And it demands it to be declared in expected array, and author with not be able to put nonsense there, as expected array is line, column, and message match.

Reply:
That is not what 2 of 3 admins asked for in issue.

https://github.com/checkstyle/checkstyle/issues/15256#issuecomment-2231354613
> We should identify anything written as // violation as a violation and then fail with an error if we can't match it with a specific message like VIOLATION_WITH_EXPLANATION_PATTERN

https://github.com/checkstyle/checkstyle/issues/15256#issuecomment-2233441811
> Ideally we match all things violation, then figure out which it is, or throw an exception if it doesn’t match.

Reply:
excetion is placed.

>Google IT no longer has an expected array, and it uses our BDD now (right?).

it uses, and it uses messages in all places as we have numerous Checks and message is always required (not enforced automatically) as it is not clear what violation is there.

Reply:
> I think it would make more sense to remove the violation below and keep the single line violation.

please give exact code to use, I do not mind to use any there.